src/app/pages/system/general-settings/localization-form/localization-form.component.spec.ts

```diff
@@ -0,0 +1,129 @@
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { ReactiveFormsModule } from '@angular/forms';
+import { MatButtonHarness } from '@angular/material/button/testing';
+import { createComponentFactory, mockProvider, Spectator } from '@ngneat/spectator/jest';
+import { mockCall, mockWebsocket } from 'app/core/testing/utils/mock-websocket.utils';
+import { LocalizationSettings } from 'app/interfaces/localization-settings.interface';
+import { IxFormsModule } from 'app/pages/common/ix/ix-forms.module';
+import { IxFormHarness } from 'app/pages/common/ix/testing/ix-form.harness';
+import { LocalizationFormComponent } from 'app/pages/system/general-settings/localization-form/localization-form.component';
+import { LanguageService, WebSocketService } from 'app/services';
+import { IxModalService } from 'app/services/ix-modal.service';
+import { LocaleService } from 'app/services/locale.service';
+
+describe('LocalizationFormComponent', () => {
+  let spectator: Spectator<LocalizationFormComponent>;
+  let loader: HarnessLoader;
+  let ws: WebSocketService;
+  const createComponent = createComponentFactory({
+    component: LocalizationFormComponent,
+    imports: [
+      IxFormsModule,
+      ReactiveFormsModule,
+    ],
+    providers: [
+      mockWebsocket([
+        mockCall('system.general.update'),
+        mockCall('system.general.language_choices', {
```

Comment:
You are not calling `system.general.language_choices` and others directly in `LocalizationFormComponent`. I think it would be better to mock `SystemGeneralService` directly since that's what being used.
There is no reason for tests for `LocalizationFormComponent` to be aware what's going on in other places, such as `SystemGeneralService`.